node_cli/core/resources.py

```diff
@@ -123,9 +123,7 @@ def generate_resource_allocation_config(env_file, force=False) -> None:
 
 
 def update_resource_allocation(disk_device: str, env_type: str) -> None:
-    resource_allocation_config = compose_resource_allocation_config(
-        disk_device, env_type
-    )
+    resource_allocation_config = compose_resource_allocation_config(env_type)
```

Comment:
We can remove disk_device from parameters

Reply:
it's already removed